exercises/06-b/README.md

```diff
@@ -0,0 +1,317 @@
+using NServiceBus;
+
+internal class Program
+{
+    public static void Main(string[] args)
+    {
+        Console.Title = "Billing";
+
+        CreateHostBuilder(args).Build().Run();
+    }
+
+    private static IHostBuilder CreateHostBuilder(string[] args)
+    {
+        var builder = Host.CreateDefaultBuilder(args);
+        builder.UseWindowsService();
+
+        builder.ConfigureLogging((ctx, logging) =>
+        {
+            // TODO: optionally choose a custom logging library
+            // https://docs.particular.net/nservicebus/logging/#custom-logging
+
+            logging.AddConfiguration(ctx.Configuration.GetSection("Logging"));
+
+            logging.AddEventLog();
+            logging.AddConsole();
+        });
+
+        builder.UseNServiceBus(ctx =>
+        {
+            var endpointConfiguration = new EndpointConfiguration("Billing");
+
+            var transport = endpointConfiguration.UseTransport<LearningTransport>();
+
+            // TODO: optionally choose a different serializer
+            // https://docs.particular.net/nservicebus/serialization/
+            endpointConfiguration.UseSerialization<NewtonsoftSerializer>();
+            endpointConfiguration.AddDeserializer<XmlSerializer>();
+
+            var conventions = endpointConfiguration.Conventions();
+            conventions.DefiningEventsAs(
+                type => (type.Namespace != null && type.Namespace.EndsWith(".Events"))
+            );
+
+            endpointConfiguration.AuditProcessedMessagesTo("audit");
+
+            endpointConfiguration.DefineCriticalErrorAction(OnCriticalError);
+
+            var displayName = System.Net.Dns.GetHostName();
+            var identifier = StringToGuid("Billing@" + displayName);
+
+            var endpointIdentity = endpointConfiguration.UniquelyIdentifyRunningInstance();
+            endpointIdentity.UsingCustomDisplayName(displayName);
+            endpointIdentity.UsingCustomIdentifier(identifier);
+
+            return endpointConfiguration;
+        });
+
+        return builder.ConfigureServices(services => { });
+
+    }
+
+    private static async Task OnCriticalError(ICriticalErrorContext context)
+    {
+        // https://docs.particular.net/nservicebus/hosting/critical-errors
+        var fatalMessage =
+            $"The following critical error was encountered:{Environment.NewLine}{context.Error}{Environment.NewLine}Process is shutting down. StackTrace: {Environment.NewLine}{context.Exception.StackTrace}";
+        EventLog.WriteEntry(".NET Runtime", fatalMessage, EventLogEntryType.Error);
+
+        try
+        {
+            await context.Stop().ConfigureAwait(false);
+        }
+        finally
+        {
+            Environment.FailFast(fatalMessage, context.Exception);
+        }
+    }
+}
+```
+
+### Step  3
+
+Add an assembly reference to `NServiceBus.Extensions.Hosting` and `Microsoft.Extensions.Hosting.WindowsServices`.
+
+### Step 4
+
+Assign the right value to `Console.Title`.
```

Comment:
I don't get this? Maybe provide a hint on why you'd want to do this, or what it should be?